src/internal/constants.ts

```diff
@@ -8,6 +8,10 @@ export const CLASS_DRAG_IMAGE = CLASS_PREFIX + "drag-image";
 export const CLASS_DRAG_IMAGE_SNAPBACK = CLASS_PREFIX + "snapback";
 export const CLASS_DRAG_OPERATION_ICON = CLASS_PREFIX + "icon";
 
+// custom event
+export const EVENT_PREFIX = "dnd-poly-event-";
+export const EVENT_DRAG_INIT = EVENT_PREFIX + "init";
```

Comment:
Another minor nitpick, but I would like to call the event `dragstart-pending` because I think it communicates better the reason and purpose for the event.

Reply:
Yes, you are right, dragstart-pending is better.